programs/past-programs.md

```diff
@@ -3,6 +3,15 @@ layout: markdown
 hero-image: url(../assets/images/stock-images/prep.jpg)
 hero-message: Past Programs
 ---
+# Fall 2016 Workshops
+
+## Building Bridges with Pioneers in Engineering
+
+The first in our series of Fall Workshops, Building Bridges with Pioneers in Engineering took place on campus on Saturday, September 24th from 9:45 AM to 3:00 PM in O'Brien Breezeway. Students from all backgrounds and experiences had tool training, learned about prototyping, design, and participated in designing and constructing their own bridge.
```

Comment:
I would add photos to these programs. And... either eliminate the wall of text, or put the photos at the side of the text. Ideally the formatting would be... nicer haha.

Reply:
Thoughts:

- Could have blog, and simple "cards" that lead to these blog posts. And blog posts that feed to Facebook. Oh wait, we do have that huh.

- More random thoughts I'll just type out here:
- - PiE Snapchat presence?
- - PiE Facebook presence could be vastly improved?
- - Event links to Facebook event on front page?
- - Encourage other Pie Staff to comment "issues" on the website?

Sorry for the brain dump here haha